CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 1.14.x (???)
+ENHANCEMENTS
+
+* Upgraded to Terraform SDK 2.24.1. Users of Pulsar will need to make minor changes in their resource files, see below.
+
+INCOMPATIBILITIES WITH PREVIOUS VERSIONS
+
+* The `ns1_application` resource attributes `config`, `default_config` and `blended_metric_weights` are now blocks, with only one item permitted. This is due to an SDK 2.x restriction on nested structures. Existing resource files will need to be edited to remove the equals sign in the declarations of the affected stanzas, for example:
```

Comment:
this seems like it could warrant a major version bump

Reply:
Done.